projects/whats-cookin-part-one.md

```diff
@@ -265,3 +241,37 @@ You are *not expected* to test:
   * Overview of technologies used, challenges, wins, and any other reflections
   * Screenshots/gifs of your app
   * List of contributors
+* **Team productively and professionally works through challenges and conflicts to ensure all team members are able to be heard and contribute throughout the project.**
+  * Instructors are available to offer support and guidance but conversations around what *is* and what *is not* working are expected to be led by the team members themselves.
+
+### Fundamental JavaScript & Style / OOP / Fetch
+- Code is divided into logical components each with a clean, single responsibility
+- Methods use array and object prototypes, not for loops
+- DOM manipulation is not present in class components
+- Variables and methods are well named
+- Code leverages JavaScript's truthy/falsey principles
+
+### Test-Driven Development
+- Application has a robust and thorough test suite that covers all class methods and class properties.
+- Test suite is organized.
+  - Each class method/class property is tested in it's own it block.
+- All scenarios/outcomes/paths are tested for your methods including happy and sad paths.
+  - Each path is tested in it's own it block.
+- Rather than using the production data, small sample data is stored in it's own file and used for testing.
+  - Sample data has been crafted to create the scenarios needed for thorough testing.
+  - For example: If you need to test a sad path of searching for recipes with a tag that no recipes match, you need test data that creates that scenario so you can test it.
+- beforeEach hook is used to DRY up test files
+- There are no failing/pending tests upon submission
+
+### User Interface
+- The application can stand on its own to be used by an instructor without guidance from a developer on the team.
+- UI/UX is intuitive and easy to read/use
+- Helpful messaging is displayed to prevent user confusion
+  - For example: If a user searches for a recipe and finds no matching results, a message is displayed to indicated that the search worked, nothing is broken, there just aren't any matching recipes found.
+- WOW > Design is responsive across small, medium and large breakpoints.
```

Comment:
Do other rubric categories get a "WOW" criteria as well?

Reply:
I just added some notes around this to the PR description.   Mostly, no - the other categories wouldn't have WOW criteria.  Other than this responsive design bullet (which is a WOW because its beyond the scope of the project requirements), breaking out bullet points into WOW doesn't seem to fit as well for the MOD 2 projects.   Its more like, we want them to strive to meet all the outlined bullets and *how well* they are able to meet those criteria is what determines if its a WOW, Yes or Not Yet

This is definitely something that I think is worth discussing further!